ortools/linear_solver/python/linear_solver.i

```diff
@@ -31,8 +31,14 @@
 %pythonbegin %{
 import os as _os
 if hasattr(_os, 'add_dll_directory'):
-    _os.add_dll_directory(_os.getenv('SIRIUS_BIN_DIR', ''))
-    _os.add_dll_directory(_os.getenv('XPRESS_BIN_DIR', ''))
+try:
+    _os.add_dll_directory(_os.getenv('SIRIUS'))
+catch AttributeError:
+    pass
+try:
+    _os.add_dll_directory(_os.getenv('XPRESS'))
+catch AttributeError:
+    pass
```

Comment:
Do we have a log handler here ? If so, consider issuing a warning if XPRESS or SIRIUS are not found.

Reply:
No we don't.
Honestly I don't think  that we need a warning.
The alternative, but it should be a different feature/PR, is to change the way the python code is generated depending on the cmake options (w or w/o Xpress, w or w/o sirius). I have no idea how to do that at the moment.
If you wish you can create a new issue